src/REAXFF/fix_reaxff_species.cpp

```diff
@@ -387,11 +381,25 @@ void FixReaxFFSpecies::init()
     auto fixcmd = fmt::format("SPECBOND_{} all ave/atom {} {} {}", id, nevery, nrepeat, nfreq);
     for (int i = 1; i < 32; ++i) fixcmd += fmt::format(" c_SPECATOM_{}[{}]", id, i);
     f_SPECBOND = dynamic_cast<FixAveAtom *>(modify->add_fix(fixcmd));
+
+    // create a fix to point to fix_property_atom for storing clusterID
+    fixcmd = fmt::format("clusterID_{} all property/atom d_clusterID ghost yes", id);
+    f_clusterID = dynamic_cast<FixPropertyAtom *>(modify->add_fix(fixcmd));
+
+    // per-atom property for clusterID
+    int flag,cols;
+    int index1 = atom->find_custom("clusterID",flag,cols);
+    clusterID = atom->dvector[index1];
```

Comment:
If this array grows, can its pointer change such that it's better to save the value of `index1` and redefine `clusterID` as needed?

Reply:
I agree that would probably be safer